marimbabot_description/urdf/marimbabot_ur5.urdf.xacro

```diff
@@ -2,15 +2,120 @@
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro"
        name="marimbabot_ur5" > 
   
+  <xacro:arg name="r200" default="false" />
+  <xacro:arg name="include_mallet" default="true" />
+
   <!-- tams corner -->
   <xacro:include filename="$(find tams_ur5_setup_description)/urdf/tams_corner.urdf.xacro">
   <!-- <xacro:arg name="floating_table" value="$(arg floating_table)" />-->
   <!-- the above argument is ignored by xacro! if you want to use this argument, try another way -->
   </xacro:include>
-  <!-- arm -->
-  <xacro:include filename="$(find tams_ur5_description)/urdf/arm.urdf.xacro" />
+
+  <!-- fnh: was <xacro:include filename="$(find tams_ur5_description)/urdf/arm.urdf.xacro" /> -->
```

Comment:
Whats up with this comment?

Reply:
Relic of copy-over from tams-ur5 repo. Removed